desync.c

```diff
@@ -41,6 +41,7 @@
 
 int get_family(struct sockaddr *dst)
 {
+    #ifndef __NetBSD__
```

Comment:
Hiding this code for NetBSD still don't fix TTL unfortunately, because `dst->sa_family` is already different at this state. I can add getsockname method for socket family detection to `get_family` function, however it will require passing socket descriptor to this function. If you are ok with this implementation is can do it by myself.

Reply:
>  because dst->sa_family is already different at this state.

proxy.c:create_conn: `pair->in6 = dst->in6;` -> `pair->in6 = addr.in6;`

Reply:
> proxy.c:create_conn: pair->in6 = dst->in6; -> pair->in6 = addr.in6;

Yeah, with your previous commit this works on NetBSD! Thank you